backend/spec/system/front_end/answers_survey_views_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'AnswersSurvey', type: :system do
+  let!(:user) { create(:user_student) }
+  let!(:survey) { create(:survey_with_answer) }
+  let(:first_question) { survey.questions.first }
+
+  describe '#create' do
+    context 'when valid' do
+      before do
+        sign_in user
+        visit surveys_path
+        click_on 'Answer'
+      end
+
+      it 'creates answers_survey' do
+        expect(AnswersSurvey.find_by(user: user, survey: survey)).to be_present
+      end
+
+      it 'redirects to the question page' do
+        expect(page).to have_current_path question_path(first_question)
+      end
+    end
+
+    context 'when not valid' do
+      before do
+        allow(AnswersSurvey).to receive(:create!).and_raise(StandardError)
+        sign_in user
+        visit surveys_path
+        click_on 'Answer'
+      end
+
+      it 'redirects to the surveys page' do
```

Comment:
you can add a test chase to check if the message from `error: err.message` is visible. Because if we are adding this message here we should be seeing it in somewhere

Reply:
https://github.com/espoo-dev/espoo-dev/pull/403/commits/b795bac410ead7680bab8e1fca7d12ae1f00fbb2